src/test/stringifiers.test.ts

```diff
@@ -0,0 +1,129 @@
+import { stringifyType } from '..';
+import * as t from '../lib/models/types';
+
+import Assert = require('assert');
+
+const union = new t.UnionType([
+    new t.IntrinsicType('string'),
+    new t.IntrinsicType('number'),
+    new t.TupleType([
+        new t.IntrinsicType('boolean'),
+        new t.IntrinsicType('string')
+    ])
+]);
+
+describe('stringifiers', function() {
+    it('maintains the `toString` method on type models', function() {
+        Assert.equal(union.toString(), stringifyType(union));
```

Comment:
Prefer `strictEqual`.

Also looks like we only use `throws` and `equal`, can we just import what we need?

```ts
import { throws, strictEqual as equal } from 'assert'
```